src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -310,6 +310,66 @@ public void castIntFieldToDoubleWithAliasJdbcFormatGroupByTest() {
         }
     }
 
+    @Test
+    public void functionAbsInWhereClauseWithIsNullTest() {
+        JSONObject response = executeJdbcRequest(
+                "SELECT age FROM " + TEST_INDEX_ACCOUNT + " WHERE (ABS(balance) IS NULL)"
+        );
+
+        String schema_result = "{\"name\":\"age\",\"type\":\"long\"}";
+        assertEquals(response.getJSONArray("schema").get(0).toString(), schema_result);
```

Comment:
np: I found that we can make it more concise and readable like [this](https://github.com/opendistro-for-elasticsearch/sql/blob/179429ad06d241d676a9a6a1eb93038d5ad8be0a/src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/DeleteIT.java#L123) using `JSONObject.query()` here and elsewhere.

Reply:
Good suggestion! Will make that change